django_slack/api.py

```diff
@@ -1,5 +1,6 @@
 import json
 
+import six
```

Comment:
I have no real problem with PEP8 etc. but you should not mix functionality and cosmetic changes, regardless of how strong the reason or whether "it was my editor".

Feel free to make a separate PR - I would just want to see only python3 changes.